### PR TITLE
Feat: Add public roadmap blog + changelog

### PR DESCRIPTION
**Pull request by adityaoberai**
## What does this PR do?

Adds the blog announcing the public roadmap

## Test Plan

- `pnpm i && pnpm run dev`
- Visit `/blog/post/public-roadmap-announcement`

## Related PRs and Issues

No

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/blog/post/public-roadmap-announcement/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Announcing Appwrite's public roadmap
+description: We just released our product roadmap to increase transparency for our community.
```
Comment (adityaoberai) [inline src/routes/blog/post/public-roadmap-announcement/+page.markdoc:4]: @LauraDuRy need to confirm the title and description with you

File: src/routes/blog/post/public-roadmap-announcement/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+## Staying updated
+We encourage you to check the public roadmap regularly to stay updated on our progress. You can also watch the repository to receive notifications about new issues, updates, and releases. This way, you will always be in the loop about what’s happening with Appwrite.
+
+## Flexibility and future planning
+While the new roadmap offers much more flexibility and visibility into our short-term and mid-term plans, it might still miss some bigger, longer-term projects or milestones that are in the early planning stages or going through an internal or external RFC (Request for Comments) process. These initiatives require more detailed planning and consultation before they can be added to the public roadmap.
+
+# The roadmap framework
+To ensure our focus is always on the right areas, we have developed an internal framework to guide our roadmap decisions. This framework is based on the significance of our focus areas, allowing us to set clear priorities, measure progress, understand the impact, maintain balance, and foster collaboration within the team and the Appwrite community.
+
+![Roadmap framework](/images/blog/public-roadmap-announcement/framework.png)
```
Comment (adityaoberai) [inline src/routes/blog/post/public-roadmap-announcement/+page.markdoc:47]: @LauraDuRy this image was available on Figma but I didn't see it in the draft on Google Docs. Is this the right place to add it?

Comment (eldadfux) [reply src/routes/blog/post/public-roadmap-announcement/+page.markdoc:47]: The location is good 👍

File: src/routes/blog/post/public-roadmap-announcement/+page.markdoc (hunk 1 above)
Comment (LauraDuRy) [reply src/routes/blog/post/public-roadmap-announcement/+page.markdoc:4]: Same comment, not "our" ;). I will adjust.

**Review by LauraDuRy — APPROVED** [review]

**Review by LauraDuRy — APPROVED** [review]

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**